src/engine/renderer/tr_scene.cpp

```diff
@@ -700,6 +700,7 @@ qhandle_t RE_RegisterVisTest()
 			break;
 		}
 	}
+	ASSERT ( !test->registered );
```

Comment:
I would prefer to return an error code, but since I don't know which value shuold be considered invalid, I simply put an ASSERT so that at least, the code would crash instead of probably overwriting content.

Reply:
Maybe add this as a comment?

Reply:
> Due to the 0 handle confusion it looks like this feature is probably broken right now. Apparently the only way it can be activated is if there is a `misc_light_flare` or `gfx_light_flare` entity in a map. Maybe @illwieckz knows how to find a map with such an entity.

Among Unvanquished stock maps, _Parpax_ and _Station15_ have some. Among Tremulous ones, _Nexus 6_ and _Niveus_ have some of those entities as well. Among community maps, _USS Tremor_ has some.

If I'm right, the flares are obvious in _Niveus_ (if what i would name flares in games are those flare entities in map).

Note: _I know there is a flare feature in the q3map2 map compiler we don't use, I don't know if it's related or not._